src/commands/enxoo/cpq/prd/import.ts

```diff
@@ -44,10 +45,11 @@ export default class Org extends SfdxCommand {
     const products = this.flags.products;
     const b2b = this.flags.b2b;
     const dir = this.flags.dir;
+    const currencies = this.flags.currencies;
```

Comment:
Small thing, not necessary now, but it starts to be a bit long so you could use destructuring assignment here